packages/core/src/components/panel-stack2/_panel-stack2.scss

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0.
 
 @import "../../common/variables";
```

Comment:
ooc should these change to `@use` at some point?

Reply:
yeah, at some point...